packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -17,6 +17,9 @@ import 'text_theme.dart';
 import 'theme.dart';
 import 'tooltip.dart';
 
+const double _kIndicatorHeight = 64;
```

Comment:
This doesn't look right. What if the app specifies a big NavigationDestination icon?

Reply:
Thanks for your comment. I actually already considered this. 

`NavigationDestination` doesn't allow overriding this width and height indicator and the splash should match the indicator size, not the icon.  

This is how it looks on Android if I provide a bigger item icon size. 
Here you can see splash size remains the same as the indicator shape size, doesn't resize when providing bigger icon size.
![Screenshot 2022-11-17 at 11 07 50](https://user-images.githubusercontent.com/48603081/202406229-f8865f4f-f888-4ce9-99f5-7c2811978ca7.png)

This is how it looks in Flutter if we provide a bigger NavigationDestination icon.
```dart
          NavigationDestination(
            icon: Icon(Icons.explore, size: 40),
            label: 'Explore',
          ),
```
![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/48603081/202407088-e09db47b-8a66-49b5-83d3-bee69e5fb98b.gif)






### Code reference


https://github.com/flutter/flutter/blob/83d205e9873fd803e032dc65f4b50fa08568c7f2/packages/flutter/lib/src/material/navigation_bar.dart#L332-L336

Here you can see `NavigationDestination` doesn't pass width or height to `NavigationIndicator`.
![Screenshot 2022-11-17 at 11 18 28](https://user-images.githubusercontent.com/48603081/202406052-9e017bdd-96fb-4bce-8c55-22335cb66b44.png)

Reply:
Thanks for the nicely illustrated explanation.